os/kernel/Kconfig

```diff
@@ -1080,6 +1080,35 @@ config SCHED_LPWORKSTACKSIZE
 endif # SCHED_LPWORK
 endmenu # Work Queue Support
 
+menu "fault Manager"
+config FAULT_MGR
+	bool "Enable Fault Manager"
+	default n
+	depends on APP_BINARY_SEPARATION
+	---help---
+		This interface is used to manage any fault occurs in a system.
+		It will either restart app or reboot the system according to
+		the configuration.
+
+if FAULT_MGR
+config DEBUG_FAULT_MGR
+	bool "Enable Debug logs of Fault Manager"
+	default n
+	depends on DEBUG
+	---help---
+		Enables debug logs for Fault Manager interface.
+
+config FM_SYSTEM_BLOCK
```

Comment:
Let's use same naming rule.
```
config FAULT_MGR_XXX
```

Reply:
okay, I will modify the name from FM_SYSTEM_BLOCK to FAULT_MGR_SYSTEM_BLOCK